### PR TITLE
REV-148 listing details page bugs

### PR DESCRIPTION
**Pull request by IbrahimAljabrST**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-148

### Description

listing details bug fixing

### CODE REVIEW

File: src/components/agent-details/ContactAgent/contact-agent.style.js [hunk 1]
```diff
@@ -45,3 +46,8 @@ export const Label = styled.label`
     font-size: 0.75rem;
   }
 `;
+
+export const ContactButton = styled(Button)`
+  font-size: 2rem;
+  background-color: green;
```
Comment (AhmadAlShwaiki) [inline src/components/agent-details/ContactAgent/contact-agent.style.js:52]:
```suggestion
  background-color:colors.green;
```

File: src/components/create-listing/CreateListingForm/index.jsx [hunk 2]
```diff
@@ -56,9 +56,11 @@ function CreateListingForm({ date }) {
       response.listingType = response.listingTypedId;
       reset(response);
     }
+    setValue('featureIds', featureIds);
 
     if (formId) fetchData();
-  }, [formId, reset]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (AhmadAlShwaiki) [inline src/components/create-listing/CreateListingForm/index.jsx:62]:
```suggestion
```

File: src/components/create-listing/CreateListingForm/index.jsx [hunk 3]
```diff
@@ -102,6 +103,10 @@ function CreateListingForm({ date }) {
       navigate('/sign-up');
     } else {
       values.featureIds = Array.from(values?.featureIds || []);
+      values.homeArea = { sqft: values.homeArea };
```
Comment (AhmadAlShwaiki) [inline src/components/create-listing/CreateListingForm/index.jsx:106]:
```suggestion
      values.homeArea = { sqft: values?.homeArea };
```

File: src/utils/helpers.js [hunk 4]
```diff
@@ -44,6 +44,16 @@ export const generateLabelValuePairs = (data) =>
...
+/**
+ * split camel case string
+ *
+ * @param {String} string text with camel case
+ *
+ * @return {String} normal text after split
+ */
+export const splitCamelCaseString = (string) =>
+  string.replace(/([a-z])([A-Z])/g, '$1 $2');
+
```
Comment (AhmadAlShwaiki) [inline src/utils/helpers.js:56]: can call toUpperCase here if that's what you always want to covert first letter toUpper

**Review by AhmadAlShwaiki — APPROVED** [review]

**Review by mohdabed123 — APPROVED** [review]

**Pull request merged**